config/overlays/tls/kustomization.yaml

```diff
@@ -12,9 +12,11 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
 - ../../base
+- resources/manager_cert.yaml
 
 patches:
 - target:
     kind: Deployment
     name: controller-manager
-  path: patches/manager_deployment_cert.yaml
\ No newline at end of file
+  path: patches/manager_deployment_cert.yaml
+- path: patches/manager_cert_patch.yaml
```

Comment:
`manager_deployment_cert.yaml` and `manager_cert_patch.yaml` seem to patch the same deployment and same fields but in different ways. I think it would be good to standardise.